package.json

```diff
@@ -49,6 +49,8 @@
     "chai": "4.1.2",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.0",
+    "eslint": "4.19.0",
```

Comment:
And what about `config-airbnb` and `jsx-a11y` ? :)

Reply:
+1 I think the airbnb-config follows sane practices.